packages/validator/src/commands/convertCmd.ts

```diff
@@ -0,0 +1,51 @@
+import fs from 'fs';
+
+import * as utils from './utils';
+import * as logger from '../utils/logger';
+import { convert, convertJson } from '../convertor';
+import { Log, Result } from '../types';
+import { invalidConversionMessage } from '../utils/messages';
+
+const oas2ois = 'OAS2OIS.json';
+const ois2config = 'OIS2Config.json';
+
+if (process.env.npm_config_template) {
+  console.log(
+    JSON.stringify(convert(process.env.npm_config_specs || process.argv[3], process.env.npm_config_template), null, 2)
+  );
+} else {
+  const from = (process.env.npm_config_from || process.argv[2] || '').toLowerCase();
```

Comment:
It is only the specification names in which it is a bad idea to allow only lowercase/uppercase. If we allow lowercase only, then it might not be intuitive to type `ois`, but if we allow only `OIS` then it doesn't look consistent with `config`. So the solution for this is to allow user to type it as they feel comfortable, since it can't cause any trouble on our end.